index.ts

```diff
@@ -103,40 +112,46 @@ class Api {
             this.executive,
             new EndCondition(this.publisher),
             new OptionValidator(),
-            this.publisher)
+            this.publisher);
     }
 
     public initialiseSimulator(options: IOptions, parseResults: IParseResult[], messageProvider: IPublishProvider) {
 
-        this.publisher.setPublishProvider(messageProvider)
+        this.publisher.setPublishProvider(messageProvider);
 
-        this.executive.initialise(options)
+        this.executive.initialise(options);
 
-        this.simulator.initialise(options, parseResults)
+        this.simulator.initialise(options, parseResults);
     }
 
     public getAt(address: number): ICoreLocation {
-        return clone(this.core.getWithInfoAt(address))
+
+        return clone(this.core.getWithInfoAt(address));
     }
 
-    public step() : void {
-        this.simulator.step()
+    public step(steps?: number) : void {
+
+        this.simulator.step(steps);
```

Comment:
This should probably be `: number` and `return this.simulator.step(steps)` to mirror the internal API. I'm not using the retVal atm, but just for consistency sake